2364/2/modules.rb

```diff
@@ -0,0 +1,78 @@
+# Module for usefull methods
+module FunctionalMethods
```

Comment:
you should avoid this type of methods and split them to service classes 

https://www.google.by/search?q=ruby+service+class&oq=ruby+service+class&aqs=chrome..69i57j0j69i60j69i65j69i60l2.2580j0j1&sourceid=chrome&ie=UTF-8